lib/utils/telemetry/generatePayload.js

```diff
@@ -66,6 +66,15 @@ const getServiceConfig = ({ configuration, options, variableSources }) => {
     ? configuration.plugins.modules || configuration.plugins
     : [];
 
+  const resolveUniqueParamsCount = (params) => {
+    if (!isPlainObject(params)) return 0;
+    return new Set(
+      Object.values(params)
+        .map((stageParams) => Object.keys(stageParams))
```

Comment:
Let's ensure `stageParams` is an object here (we may deal with invalid configurations here)